pkg/apis/psmdb/v1/perconaservermongodbbackup_types.go

```diff
@@ -45,6 +45,7 @@ type PerconaServerMongoDBBackupStatus struct {
 	Azure          *BackupStorageAzureSpec `json:"azure,omitempty"`
 	ReplsetNames   []string                `json:"replsetNames,omitempty"`
 	PBMname        string                  `json:"pbmName,omitempty"`
+	PBMpod         string                  `json:"pbmPod,omitempty"`
```

Comment:
`PBMPod`?

Reply:
I know, I know, the idea was to be consistent with `PBMname`, but maybe it shouldn't be in this case.